packages/control/chargepoint/chargepoint.py

```diff
@@ -282,8 +282,10 @@ def _set_values_at_start(self):
                       self.data.set.plug_time)
 
     def remember_previous_values(self):
+        self.data.get.charge_state_prev = self.data.get.charge_state
```

Comment:
```suggestion
        self.data.set.charge_state_prev = self.data.get.charge_state
```
Der Wert wird nicht aus dem Ladepunkt ausgelesen, daher bitte analog zu plug_state_prev nach Set verschieben.